order/order.html

```diff
@@ -8,6 +8,68 @@
     <link rel="stylesheet" href="./order.css">
 </head>
 <body>
-    
+    <header>
+        <img src="../img/Icon Circle.svg" class="backicon" alt="icon">
+        <h1 class="titlepage">Your Orders</h1>
+    </header>
+    <div class="container">
+        <div class="discount1">
+            <h1 class="dis-1-title">30% OFF</h1>
+            <p class="dis-1-details">Use code 87412 at checkout</p>
+        </div>
+        <!--<div class="discount2">
+            <h1 class="dis-2-title">30% OFF</h1>
+            <p class="dis-2-details">Use code 87412 at checkout</p>
+        </div>-->
+    </div>
+    <div>
+        <section class="coffee-latte">
+          <div class="pictures">
+            <img src="../img/coffe-latte.svg" class="latte" alt="coffee latte">
+          </div>
+          <div class="coffee-latte-details">
+            <p class="coffee-latte-title"> Caffe Latte</p>
+            <p class="Milk-and-espresso">Milk and espresso</p>
+            <p class="tall">TALL</p>
+            <p class="price1">$ 5.99</p> 
+          </div>
+          <div class="amount-pic">
+            <img src="../img/Amount.svg" class="amount1" alt="amount">
+          </div>
+        </section>
+        <section class="coffee-latte">
+            <div class="pictures">
+              <img src="../img/caffe-americano 1.svg" class="latte" alt="coffee latte">
+            </div>
+            <div class="coffee-latte-details">
+              <p class="coffee-latte-title"> Original Americano</p>
+              <p class="Milk-and-espresso">Classic blend</p>
+              <p class="short">SHORT</p>
+              <p class="price1">$ 4.99</p> 
+            </div>
+            <div class="amount-pic1">
+              <img src="../img/Amount.svg" class="amount1" alt="amount">
+            </div>
+          </section>
+          <section class="coffee-latte">
+            <div class="pictures">
+              <img src="../img/mocca.svg" class="latte" alt="mocca coffee">
+            </div>
+            <div class="coffee-latte-details">
+              <p class="coffee-latte-title"> Peppermint Mocha</p>
+              <p class="Milk-and-espresso">Real chocolate and mint</p>
+              <p class="grande">GRANDE</p>
+              <p class="price1">$ 18.98</p> 
+            </div>
+            <div class="amount-pic1">
+              <img src="../img/Amount2.svg" class="amount1" alt="amount">
+            </div>
+          </section>
+    </div>
+    <hr>
+    <div class="totalnumber">
+        <p class="total">Total</p>
+        <p class="number">$ 29.96</p>
+    </div>
```

Comment:
You miss last two elements in the page , check the Figma design

Reply:
I added them in the issue #34